pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```
@@ -45,6 +47,9 @@ let
       HZ = freeform "250";
       HZ_250 = yes;
       HZ_1000 = no;
+
+      # Disable writeback throttling by default 
```

```suggestion
      # Disable writeback throttling by default
```
I guess CI complains about this whitespace

---

Please make sure your editor properly enforces editorconfig!